web/src/test/java/com/graphhopper/http/SpatialRulesTest.java

```diff
@@ -49,11 +49,9 @@
 
     private static GraphHopperServerConfiguration createConfig() {
         GraphHopperServerConfiguration config = new GraphHopperServerTestConfiguration();
-        // The EncodedValue "country" requires the setting "spatial_rules.borders_directory" as "country" does not load via DefaultTagParserFactory
-        // TODO should we automatically detect this somehow and include a default country file?
         config.getGraphHopperConfiguration().
                 putObject("graph.flag_encoders", "car").
-                putObject("graph.encoded_values", "country,road_environment,road_class,road_access,max_speed").
+                putObject("graph.encoded_values", "spatial_rule,road_environment,road_class,road_access,max_speed").
```

Comment:
Maybe worth mentioning this in the changelog?

Reply:
Seems like a good idea. I wasn't quite sure about the renaming but it makes no sense to stick with `country` as users could encode arbitrary things here.